### DISCUSSION TITLE
skipBeacon update

### DISCUSSION THREAD

**Discussion by AbramovychYurii** [gilbarbara/react-joyride#690] [Ideas]
Hello everyone. Faced with such a problem, I need to display a beacon after each step, but if I disable the property "continuous" it will not works, because the "next" button will change to "close". And to change that I need to add my own component and handle events, it seems difficult for such small changes. It might be a good idea to add a property that would allow you to display a beacon after each step. Now blocked due to that `index > 0` in [Step component](https://github.com/gilbarbara/react-joyride/blob/master/src/components/Step.js) (line 110)

**Comment by gilbarbara** [comment]
Does the tour work as you expect with `continuos` set to false?
The only problem is the label being `Close` instead of `Next`?

You can change the label with the [locale](https://docs.react-joyride.com/props) prop.

If need more customizations you can use your own component, you can read more about it in the docs.

[s]

**Reply by AbramovychYurii** [reply]
Yes, but it requires additional custom components and settings. It would be better if it was easier and depended only on one property.

**Reply by AbramovychYurii** [reply]
It seems to be a fairly common case (to show a beacon after each step), when you need not only to highlight the elements, but also to wait for some user action to go to the next step.

**Comment by gilbarbara** [comment]
@AbramovichCp 
Please post a codesandbox with a minimal reproducible demo to really understand what you are trying to achieve.
From what I understood just changing the label would work...
